packages/dashboard-frontend/src/pages/UserPreferences/GitServicesTab/__tests__/__snapshots__/index.spec.tsx.snap

```diff
@@ -249,6 +249,283 @@ Array [
             onMouseEnter={[Function]}
           >
             GitLab
+            <span
+              data-testid="gitlab-unsupported-automatic-revoke-tooltip"
+              data-tip="Provided API does not support the automatic token revoke. You can revoke it manually."
```

Comment:
Can we add URL in the tooltip that would be `GitLab` specific?

Reply:
I have fixed it.